### PR TITLE
update documentation

### PR DESCRIPTION
**Pull request by manu-akw**
fix decoding of password

### CODE REVIEW

File: website/docs/index.html.markdown [hunk 1]
```diff
@@ -134,7 +134,7 @@ data "aws_secretsmanager_secret_version" "postgres_password" {
 
 provider "postgresql" {
    [...]
-   password = data.aws_secretsmanager_secret_version.postgres_password.secret_string
+   password = jsondecode(data.aws_secretsmanager_secret_version.postgres_password.secret_string)["password"]
```
Comment (cyrilgdn) [inline website/docs/index.html.markdown:137]: Correct me if I'm wrong but this seems to depends of what you put in the secret_string no?

I mean one could put the password directly and not a JSON containing a password field.

Comment (manu-akw) [reply website/docs/index.html.markdown:137]: But if you use the AWS Secrets Manager, the secrets are always stored as key/value pairs, so there is no way around the JSON format.

Comment (cyrilgdn) [reply website/docs/index.html.markdown:137]: > the secrets are always stored as key/value pairs

Actually no, if you define the secret with the key/value tab in the AWS console, it indeed generates JSON, but the `SecretString` is a text in which you can put what you want.
In the AWS console, there's even a "plaintext" tab in which you can enter a raw string, e.g.:

image

But as your example could indeed be a typical setup (many users will set a JSON), I'm ok to merge this example :+1:

**Review by cyrilgdn — APPROVED** [review]

File: website/docs/index.html.markdown (hunk 1 above)
Comment (manu-akw) [reply website/docs/index.html.markdown:137]: Well, I took an extra look at the console yesterday, but somehow overlooked this option. Sorry, you're right👍

Just a small addition: if you create an RDS in AWS with Terraform, it is possible to set the attribute "manage_master_user_password" to true, then the credentials are automatically managed by the RDS itself in the Secrets Manager. And then they are automatically saved in JSON format. 
That was the actual reason for this PR, coming from this setup.

Comment (cyrilgdn) [reply website/docs/index.html.markdown:137]: > then the credentials are automatically managed by the RDS itself in the Secrets Manager. And then they are automatically saved in JSON format

@manu-akw Thanks, I didn't know that :+1:

**Pull request merged**